star_triangle.py

```diff
@@ -0,0 +1,18 @@
+BASE = 15
+HEIGHT = 8
+FILL_SYM = ' '
+OUT_TEMPLATE = '*'
+BORDER_WIDTH = len(OUT_TEMPLATE)
+
+
+def draw_triangle():
+    half_base = BASE // 2
+    for i in range(HEIGHT):
+        print((FILL_SYM * (half_base * BORDER_WIDTH - i * BORDER_WIDTH)) + OUT_TEMPLATE * (i + 1) + OUT_TEMPLATE * i)
```

Comment:
1. Violation of logic / output  rule:
extract calc to the separate var

Reply:
I don't understand how to perform it, please help me with this.